tests/harvest/test_harvest_main.py

```
@@ -0,0 +1,34 @@
+import pytest
+
+from sema.harvest.__main__ import _main
+
+log = logging.getLogger(__name__)
+
+TEST_CONFIG_FOLDER = Path(__file__).parent / "config"
+TEST_INPUT_FOLDER = Path(__file__).parent / "inputs"
+
+
+@pytest.mark.usefixtures("outpath", "store_info_sets")
+def test_main(outpath: Path, store_info_sets: tuple):
```

Ruff

Missing return type annotation for public function `test_main`

Add return type annotation: `None`

(ANN201)